aspnet/mvc/views/forms/sample/final/ViewModels/CountryViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNet.Mvc.Rendering;
+using System.Collections.Generic;
+
+public class CountryViewModel
+{
+    public string Country { get; set; }
+
+    public List<SelectListItem> Countries { get; private set; }
+
+    public CountryViewModel()
```

Comment:
On second thought, use more of C#6 and get rid of unnecessary `private set;` as well as the constructors in these view models e.g.
``` c#
public List<SelectListItem> Countries { get; } = new List<SelectListItem>
{
    new SelectListItem { Value = "MX", Text = "Mexico" },
    new SelectListItem { Value = "CA", Text = "Canada" },
    new SelectListItem { Value = "US", Text = "USA"  },
};
```